src/github.com/getlantern/flashlight/feed/feed.go

```diff
@@ -297,3 +289,20 @@ func determineLocale(defaultLocale string) string {
 	}
 	return defaultLocale
 }
+
+func getHTTPClient() (*http.Client, error) {
+	var err error
+	httpClientMutex.Lock()
+	if _httpClient == nil {
+		_httpClient, err = proxied.ChainedNonPersistent("")
```

Comment:
The cost is not in creating an HTTP client, the cost is in that HTTP client opening connections to our proxy.  Previously, we did that for every request made by GetFeed because it created a non-persistent http client every time it was called.  Now, we have a single persistent HTTP client that can reuse connections within a single call to GetFeed and across calls to GetFeed.